### ISSUE TITLE
Code

### ISSUE THREAD

**Issue by PROFESSORRQ** [vh-diazr/dehazing-LocallyAdaptive#2]
Can I run this code on my laptop? 
Because I am trying again and again and some error is coming.

**Comment by vh-diazr** [comment]
Yes, you just have to make sure to meet the prerequisites outlined in the README file.
